docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+
+Toch zijn er nog enkele openstaande vragen te beantwoorden. Voor een deel is dit al af te leiden uit de stippellijnen in bovenstaande plaat:
+
+### 1.4.1. Hoe verhoudt melding/verzoek zich tot een zaak?
+
+In bovenstaande figuur is de relatie tussen Melding/Aanvraag (Verzoek) en Zaak als een stippellijn getekend. Dit omdat Verzoek niet in het RGBZ voorkomt (zat wel in GFO Zaken) en we dus niet weten hoe Verzoek en Zaak zich tot elkaar verhouden. Essentie van de relatie lijkt te zijn dat een verzoek of melding een op zich zelf staand ‘iets’ is en behandeld wordt als zaak. Één verzoek kan leiden tot meerdere zaken. Meerdere Verzoeken kunnen ook leiden tot één Zaak?
+Documenten bij een Verzoek worden bij het Verzoek vastgelegd. Later ook rechtstreeks gerelateerd aan de zaak?
+
+### 1.4.2. Waar slaan we procesinformatie op?
+
+Het is nog onduidelijk waar we in een gegevenslandschap “Procesinformatie” willen opslaan. In bovenstande figuur is dit – samen met procesdefinitiegegevens - gepositioneerd in de Zaakafhandelcomponent (net als nu in de diverse Taaksystemen).
+Dit is echter (mogelijk) in strijd met het uitgangspunt in Common ground dat functionaliteit en gegevens van elkaar gescheiden moeten zijn. Procesgegevens zijn immers ook gegevens.
+De procesdefinitie is in veel gevallen zelfs impliciet (hard gecodeeerde flow tussen schermen); dat valt dan niet eens los te knippen.
+
+Conclusie: hier is sprake van een glijdende schaal. Je wilt liefst alle domeingegevens en zo veel mogelijk procesgegevens uit de applicaties halen en via een gestandaardiseerde APi ontsluiten. We moeten echter wel oppassen dat we hier te ver in willen gaan en het paard daarmee achter de wagen spannen. In de praktijk zal dit voor veel bestaande taaksystemen een groeipad worden..
+
+Er is één maar. Het delen van procesinformatie is nodig om het handelen als overheid te verantwoorden: het moet dus gearchiveerd worden. Wie heeft op welke datum welke stap gezet? Welke procesflow (en keuzes) is er doorlopen? Welke informatie was wanneer beschikbaar? Hoe luidde de (deel)berekening die heeft geleid tot een bepaald resultaat? Dit is informatie die niet in één van de andere registraties past (tenzij als logfile in PDF in de DRC) en voor het grootste deel niet in de ZRC wordt vastgelegd (die kent meestal maar één behandelaar en vooral enkel het proces op hoofdlijnen: de Statussen).
+Dit pleit voor een procesverloopregistratie, waarin op een zo gestandaardiseerde manier als mogelijk informatie over het verloop van alle processen wordt bijgehouden. Probleem is echter dat het ene proces het andere niet is en het erg moeilijk is dit gestandaardiseerd te doen (een zaak probeert dat feitelijk ook, en deze kent niet voor niets enkel statussen). Dus wellicht krijg je dan een subsiedieprocesverloopregistratie, een vergunningprocesverloopregistratie, een burgerzakenprocesverloopregistratie, … Dat is dan weer niet aanlokkelijk.
```

Comment:
Ik denk dat je dat niet kunt standaardiseren. De ZAC, generiek en specifiek, zijn vrij om procesondersteuning op hun eigen manier vorm te geven. Ik denk dat daar ook de vrijheid van leveranciers zit om het verschil te maken. Je zegt het zelf eigenlijk ook: een zaak is al een neerslag van het gelopen proces.

Reply:
mee eens. alleen bevat ZAAK wellicht onvoldoende informatie om het handelen van de overheid te verantwoorden..